client/js/screens/Campaigns/styles.js

```diff
@@ -1,3 +1,88 @@
 import {StyleSheet} from 'react-native';
-const styles = StyleSheet.create({});
+const styles = StyleSheet.create({
+  xxxIcon: {
+    resizeMode: 'contain',
+    height: 30,
+    alignSelf: 'flex-end',
+  },
+  categoryImage: {
+    height: 60,
+    width: 100,
+    resizeMode: 'contain',
+    // alignSelf: 'flex-start',
+  },
+  campaignsHeader: {
+    justifyContent: 'space-between',
+    flexDirection: 'row',
+    alignItems: 'center',
+    paddingTop: 10,
+    marginBottom: 10,
+    marginRight: 60,
+    marginLeft: 20,
+  },
+  categoryVideo: {
+    width: 130,
+    height: 80,
```

Comment:
use %s.

Reply:
I changed the height to percent but when I change the width. My entire app crashes.